vulnfeeds/cmd/pypi/main.go

```diff
@@ -78,6 +78,7 @@ func main() {
 	pypiLinksPath := flag.String("pypi_links", "", "Path to pypi_links.json.")
 	pypiVersionsPath := flag.String("pypi_versions", "", "Path to pypi_versions.json.")
 	falsePositivesPath := flag.String("false_positives", "", "Path to false positives file.")
+	withoutNotes := flag.Bool("without_notes", false, "Output vulnerabilities without notes only.")
```

Comment:
maybe skip_notes ?

Reply:
See other comment above. I think skip_notes implies something different here.

Reply:
ignore this please, i didnt understand.